gadgets/profile_blockio/README.mdx

```diff
@@ -0,0 +1,210 @@
+
+        ```bash
+        $ kubectl gadget run profile_blockio:latest --node minikube-docker
+        ```
+
+        It will start to display the I/O latency distribution as follows:
+
+        ```bash
+        latency
+              µs               : count    distribution
+               0 -> 1          : 0        |                                        |
+               1 -> 2          : 0        |                                        |
+               2 -> 4          : 0        |                                        |
+               4 -> 8          : 1        |*                                       |
+               8 -> 16         : 21       |***************************             |
+              16 -> 32         : 17       |*********************                   |
+              32 -> 64         : 12       |***************                         |
+              64 -> 128        : 0        |                                        |
+             128 -> 256        : 0        |                                        |
+             256 -> 512        : 0        |                                        |
+             512 -> 1024       : 21       |***************************             |
+            1024 -> 2048       : 3        |***                                     |
+            2048 -> 4096       : 31       |****************************************|
+            4096 -> 8192       : 0        |                                        |
+            8192 -> 16384      : 0        |                                        |
+           16384 -> 32768      : 0        |                                        |
+           32768 -> 65536      : 0        |                                        |
+           65536 -> 131072     : 0        |                                        |
+          131072 -> 262144     : 0        |                                        |
+          262144 -> 524288     : 0        |                                        |
+          524288 -> 1048576    : 0        |                                        |
+         1048576 -> 2097152    : 0        |                                        |
+         2097152 -> 4194304    : 0        |                                        |
+         4194304 -> 8388608    : 0        |                                        |
+         8388608 -> 16777216   : 0        |                                        |
+        16777216 -> 33554432   : 0        |                                        |
+        33554432 -> 67108864   : 0        |                                        |
+        ```
+
+        Now, let's increase the I/O operations using the stress tool:
+
+        ```bash
+        # Start by creating our testing namespace
+        $ kubectl create ns test-biolatency
+
+        # Run stress with 1 worker that will generate I/O operations
+        $ kubectl run --restart=Never --image=polinux/stress stress-io -n test-biolatency -- stress --io 1
```

Comment:
Shouln't we use `-hdd` instead?
```suggestion
        $ kubectl run --restart=Never --image=polinux/stress stress-io -n test-biolatency -- stress --hdd 1
```

Reply:
This is coming from: https://github.com/inspektor-gadget/inspektor-gadget/blob/main/docs/builtin-gadgets/profile/block-io.md . I will prefer to keep it as it is for now and can improve it follow up PR?